lib/Yam/Agama/Pom/GrubMenuAgamaPage.pm

```diff
@@ -15,13 +15,14 @@ sub new {
     my ($class, $args) = @_;
     return bless {
         grub_menu_base => $args->{grub_menu_base},
-        tag_first_entry_highlighted => 'grub-menu-agama-installer-highlighted',
+        tag_agama_installer_highlighted => 'grub-menu-agama-installer-highlighted',
```

Comment:
I would recommend here to be more generic, `tag_install_product_entry_highlighted`.